### ISSUE TITLE
error signing up for account

### ISSUE THREAD

**Issue by Markl121** [Codewars/codewars.com#1516]
![image](https://user-images.githubusercontent.com/36147343/43104087-2ec915d6-8e9e-11e8-8eda-b1037096d34f.png)

Hello,

I tried to sign up for Codewar and the following error occurred. I also tried to use the link Github link, but an error message saying I was marked as span displayed. Please help

**Comment by anter69** [comment]
@kazk Duplicate of #1512 and many others. Could be closed.

**Comment by kazk** [comment]
Thanks, I'll close #1512 since this one has more reactions.

**Comment by codeidoscope** [comment]
I thought I'd post on here to report the behaviour I encountered when trying to sign up. I have two Github accounts, one that is already linked to Codewars, and one that was not.

Using Chrome, I tried to create an account with my newest Github account and accidentally tried to reuse the username I had set for my existing account (I didn't realise I was already using it). 

Although I was not able to use the site for the most part (it would display pages but I would not be logged in), I did somehow manage to reach the Account Setting page and update my skills on there (keep that in mind for later).

I encountered the error message posted by OP every time I tried to sign up (which I did two or three times yesterday and today), and only when I tried to sign up on Safari (in order to see if it was a browser issue) did the website inform me that the username was already in use.

Keen to keep my username on Codewars, I decided to log into my existing Codewars account, unlink my old Github account and relink my new Github one. When I got to my existing Codewars account's Account Settings page - and before I had made any changes to it or unlinked/relinked accounts - my email, clan and skills had already been updated with the data that I was using for the creation of my new account.

With the exception of the username, the accounts had nothing in common (different clans, email addresses, etc.) - I think it might be worth investigating is this cannibalisation happens for other people.

I've posted this here as this is the issue I thought I was originally faced with, but I felt the behaviour I eventually encountered might provide you with some insights as to what the issue may be. Let me know if you would prefer me to open a separate issue.

**Comment by paknauer** [comment]
@codeidoscope This is interesting and would explain what happened. At the beginning I had the same message posted by the OP. With your information I think I tried to use an existing username and now I seem to have updated the email address of the other account. Trying it with another (very unlikely to be duplicate) username I got an error message at the email adress which said "is already taken". I wonder if the other user will ever find out that I changed his/her email address to mine (if it went that way). Using another email address it worked.

**Comment by Enigma-Games** [comment]
I have the exact same problem as OP. Any fixes yet?

**Comment by codeidoscope** [comment]
@Enigma-Games Try another username that you know won't exist - it looks like it might be silently cannibalising someone else's existing account.

**Comment by CalebBurton** [comment]
Same issue for me. Tried a few extremely unique (20 random alphanumeric characters) usernames and they all gave the same error after timing out. I tried signing up with just an email/username/password, no GitHub account linking.

Using a super common username resulted in me seeing a 500 error **but also** apparently being logged into someone else's account dashboard. I could view things, but when I clicked on anything it would ask me to sign in and the credentials I used when signing up didn't work. Using a unique username resulted in a 500 error **without** showing me logged into anyone's account.

Console logs are blank. Issue persists across the most up-to-date versions of Firefox, Chrome, and Chrome Incognito with no extensions enabled.
